components/inputs/docs/input-radio.md

```diff
@@ -28,127 +23,87 @@ As a result, we have to apply styles to native radio inputs.
 <!-- docs: end dos -->
 
 <!-- docs: start donts -->
-* Don’t use two radio buttons if a single checkbox works better
+* Don’t use two radio inputs if a single checkbox works better
 * Don’t use for triggering an immediate action. Notable exceptions are forms that autosave with clear indication and as a trigger for progressive disclosure on traditional forms, so long as users are made aware that new options have been made available
 <!-- docs: end donts -->
 <!-- docs: end best practices -->
 
-## Radio Inputs With Labels
+## Radio Input Group [d2l-input-radio-group]
 
-The simplest way to apply radio styles is to use the `d2l-input-radio-label` CSS class on a `<label>` element that wraps the input.
+The group is a required parent of `<d2l-input-radio>`. It internally renders a `<fieldset>` and `<legend>` with the provided `label`, which gives additional accessibility context. The label can be hidden visually if desired.
```

Comment:
Same as above.

Reply:
I think the same justification applies -- that devs are told to wrap groups of checkboxes or radios in fieldsets, but they need to be aware _not_ to do that because this component does that for them.